lib/vsc/ldap/utils.py

```diff
@@ -67,13 +67,13 @@ def __init__(self, url=None, password=None, connection_dn=None, validation_metho
 
         self.log = getLogger(self.__class__.__name__)
 
-    def tls_settings(self, url):
+    def tls_settings(self):
         """
         Set up the LDAP for a TLS connection.
         """
-
-        # FIXME:maybe we want some default configuration here?
-        pass
+        ldap.set_option(ldap.OPT_X_TLS_DEMAND, True)
+        ldap.set_option(ldap.OPT_X_TLS_REQUIRE_CERT, True)
+        ldap.set_option(ldap.OPT_X_TLS_NEWCTX, 0)
```

Comment:
what is this?

Reply:
making sure the connection actually starts using TLS settings, e.g., don't use what was set before.

Reply:
otherwise these settings are ignored